base/src/test/java/uk/ac/ebi/atlas/solr/analytics/query/AnalyticsQueryClientIT.java

```diff
@@ -44,127 +51,157 @@
+        assertThat(queryMade).doesNotContain("keyword_");
+        assertThat(queryMade).contains("identifier_search");
+        assertThat(queryMade).contains("zinc finger");
 
-        assertThat(queryMade, containsString("conditions_search"));
-        assertThat(queryMade, containsString("liver"));
+        assertThat(queryMade).contains("conditions_search");
+        assertThat(queryMade).contains("liver");
     }
 
     @Test
-    public void queryConditionSearchOrIdentifierSearchIncludesTheQueryStringTwiceForQueriesWithNoCategory(){
+    void queryConditionSearchOrIdentifierSearchIncludesTheQueryStringTwiceForQueriesWithNoCategory() {
         String queryMade = subject.queryBuilder()
                 .bioentityIdentifierFacets(-1)
                 .queryIdentifierOrConditionsSearch(SemanticQuery.create("tasty pancake"))
                 .fetch();
 
-        assertThat(queryMade, not(containsString("keyword_"))); //two words so this is not a keyword
-        assertThat(queryMade, containsString("identifier_search"));
-        assertThat(queryMade, containsString("conditions_search"));
-        assertThat(queryMade.split("tasty pancake").length, greaterThan(2));
+        assertThat(queryMade).doesNotContain("keyword_"); //two words so this is not a keyword
+        assertThat(queryMade).contains("identifier_search");
+        assertThat(queryMade).contains("conditions_search");
+        assertThat(queryMade.split("tasty pancake").length).isGreaterThan(2);
     }
+    
+    @Test
+    void baselineFacetsOnlyReturnsBaselineExperiments() {
+        String queryResponse = goodSubject.queryBuilder()
+                .baselineFacets()
+                .queryIdentifierOrConditionsSearch(SemanticQuery.create("lung"))
+                .fetch();
 
+        List<String> experimentTypes =
+                JsonPath.using(Configuration.defaultConfiguration().addOptions(Option.SUPPRESS_EXCEPTIONS))
+                        .parse(queryResponse)
+                        .read("$.facets.experimentType.buckets[*].val");
 
-    class TestableAnalyticsQueryClient extends AnalyticsQueryClient {
+        // If we want to go to an extra level of correctness we can get the experiment type from the DB, but if the
+        // type isn’t right in Solr our problems are going to be bigger than this test
+        assertThat(experimentTypes)
+                .isNotEmpty()
+                .allMatch(type -> ExperimentType.get(type).isBaseline());
+    }
 
-        public TestableAnalyticsQueryClient(RestTemplate restTemplate, String solrBaseUrl, Resource baselineFacetsQueryJSON, Resource differentialFacetsQueryJSON,Resource experimentTypesQueryJson, Resource bioentityIdentifiersQueryJson) {
-            super(restTemplate, solrBaseUrl, baselineFacetsQueryJSON, differentialFacetsQueryJSON, experimentTypesQueryJson, bioentityIdentifiersQueryJson);
+    class TestableAnalyticsQueryClient extends AnalyticsQueryClient {
```

Comment:
Hmm not sure this is such a good idea... AnalyticsQueryClient should be "testable" already! Out of scope for this PR though.